test/integration/apiserver/apply/apply_test.go

```diff
@@ -367,6 +386,81 @@ func TestApplyUpdateApplyConflictForced(t *testing.T) {
 	}
 }
 
+// TestUpdateApplyConflict tests that applying to an object, which wasn't created by apply, will give conflicts
+func TestUpdateApplyConflict(t *testing.T) {
```

Comment:
Can we move this test to `apiserver/pkg/endpoints/handlers/fieldmanager` instead? I think the code that we're testing completely lives in that folder